.github/config/xb1.json

```diff
@@ -3,6 +3,9 @@
   "__comment" : "TODO: Deploy a runner and change this",
   "docker_runner_service": "runner-xb1",
   "runner_tag": "xb1",
+  "on_device_test": {
+    "enabled": true
+  },
```

Comment:
You also need to configure the test shards. Otherwise github won't generate the test runner jobs.

```
  "on_device_test": {
    "enabled": true,
    "tests": [
      "0",
      "1",
      "2",
      "3",
      "black_box_test"
    ]
```

Reply:
I included black_box_tests here since it is in the android config. I don't know if we should be running that suite on xb1.

Reply:
since this is a feature/ branch feel free to modify the code yourself. 

I'm not too familiar with this code for adding unit tests

Reply:
Then just add the unit test shards for now.  The block should look like this:

```
  "on_device_test": {
    "enabled": true,
    "tests": [
      "0",
      "1",
      "2",
      "3"
    ]
```